hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/execution/init/ExecutionPlanBuilder.java

```diff
@@ -87,8 +88,8 @@ private ExecutionPlanBuilder() {
                     e -> vertexIdMap.get(e.getSourceName()), isJobDistributed);
             final List<EdgeDef> outbound = toEdgeDefs(dag.getOutboundEdges(vertex.getName()), defaultEdgeConfig,
                     e -> vertexIdMap.get(e.getDestName()), isJobDistributed);
-            final ILogger logger = nodeEngine.getLogger(String.format("%s.%s#ProcessorMetaSupplier",
-                    metaSupplier.getClass().getName(), vertex.getName()));
+            String prefix = String.format("%s#ProcessorMetaSupplier", vertex.getName());
```

Comment:
This would be easier to read:
```suggestion
            String prefix = vertex.getName() + "#ProcessorMetaSupplier";
```